codeforboston/maple
# 1607 add llm genrated tag

Summary

A functional LLM tags Hierarchical filter

Checklist

1. A new Instantsearch Facet that filters on Tags

2. This new content should be toggleable by a new flag in FeatureFlags

Screenshots
<img width="1432" alt="Screenshot 2025-01-05 at 1 13 05 PM" src="https://github.com/user-attachments/assets/567f46d5-eb8b-4213-b56d-809d24037e02" />


Known issues
To improve the functionality we can add search bar to the filer.

Steps to test/reproduce

Go to the browse bill page.
Click on "+" to expand the tag
Click on subtitle to do the filter and check if tag shows on the top of search result.
